src/components/Field/Field.tsx

```diff
@@ -47,19 +43,27 @@ export class Field extends React.PureComponent<FieldProps> {
       action,
       onAction,
       disabled,
-      kind,
-      fitContent,
       ...rest
     } = this.props
     const isAddress = this.isAddress()
-    const icon = error && !isAddress ? 'warning circle' : void 0
-    const classes = classnames('dcl field', kind, {
-      error,
-      disabled,
-      address: isAddress,
-      resizable: fitContent,
-      ['has-label']: label
-    })
+    let classes = 'dcl field'
+    let icon
+
+    if (error) {
+      classes += ' error'
+      if (!isAddress) {
+        icon = 'warning circle'
+      }
+    }
+    if (isAddress) {
+      classes += ' address'
+    }
+    if (disabled) {
+      classes += ' disabled'
+    }
+    if (label) {
+      classes += ' has-label'
+    }
```

Comment:
Would you mind using `classnames`?
You can conditionally add classes using a map:
```
{
 address: isAddress,
 disabled,
 resizable: fitContent,
 ['has-label']: label
}
```

Reply:
sorry it was a version error i dindn't want to change this